CHANGELOG.md

```diff
@@ -12,45 +12,55 @@ be also set to `true`. Otherwise, the template engine will log an error but then
 restart its internal runner. [[GH-11775](https://github.com/hashicorp/vault/pull/11775)]
 * agent: Update to use IAM Service Account Credentials endpoint for signing JWTs
 when using GCP Auto-Auth method [[GH-11473](https://github.com/hashicorp/vault/pull/11473)]
+* core: License/EULA changes that ensure the presence of a valid HashiCorp license to
+start Vault. More information is available in the [Vault License FAQ](https://www.vaultproject.io/docs/enterprise/license/faqs)
 * go: Update to Go 1.16.5 [[GH-11802](https://github.com/hashicorp/vault/pull/11802)]
 
 FEATURES:
 
+* **GCP Secrets Engine Static Accounts**: Adds ability to use existing service accounts for generation
+  of service account keys and access tokens. [[GH-12023](https://github.com/hashicorp/vault/pull/12023)]
+* **Key Management Secrets Engine (Enterprise)**: Adds general availability for distributing and managing keys in AWS KMS. [[GH-11958](https://github.com/hashicorp/vault/pull/11958)]
+* **License Autoloading**: Licenses may now be automatically loaded from the environment or disk.
```

Comment:
Should this also need a ` (Enterprise)`?

Reply:
Updated